src/Tizen.NUI.Components/Utils/StyleManager.cs

```diff
@@ -89,26 +60,48 @@ public string Theme
         {
             get
             {
-                return currentThemeName;
+                return ThemeManager.CurrentTheme.Id;
```

Comment:
on mobile or tv profile, ThemeManager.CurrentTheme is null.
it causes crash System.NullReferenceException.